swagger.yaml

```diff
@@ -1,12 +1,9 @@
 swagger: '2.0'
-host: sinopia.io/
```

Comment:
this was an attempt to allow specifying host name to make testing easier.  not sure if it's the right approach, but appeared to be suggested by swagger docs:  https://swagger.io/docs/specification/2-0/api-host-and-base-path/